parity-clib/examples/cpp/main.cpp

```diff
@@ -16,22 +16,23 @@
 
 #include <chrono>
 #include <parity.h>
+#include <parity.hpp>
 #include <regex>
 #include <string>
 #include <thread>
 
-void* parity_run(std::vector<const char*>);
-int parity_subscribe_to_websocket(void*);
-int parity_rpc_queries(void*);
+static parity_ethereum *parity_run(std::vector<const char *>);
+static int parity_subscribe_to_websocket(parity_ethereum *);
+static int parity_rpc_queries(parity_ethereum *);
```

Comment:
We generally should avoid using non-fixed sized types such as `int` and instead use types defined in https://en.cppreference.com/w/cpp/header/cstdint or https://en.cppreference.com/w/cpp/header/cstddef